### PR TITLE
Update README-ko.md file

### PR DESCRIPTION
**Pull request by suhyenim**
## Proposed changes

Translate README.md file into Korean.

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [x] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

**Review by namkyu1999 — CHANGES_REQUESTED** [review]
Could you please check the reviews below? rest looks good to me :)

File: translations/README-ko.md [hunk 1]
```diff
@@ -1,98 +1,162 @@
...
 
-- **ChaosEngine**: 쿠버네티스 애플리케이션 또는 쿠버네티스 노드를 ChaosExperiment에 연결하는 리소스입니다. ChaosEngine은 리트머스의 카오스-오퍼레이터(Chaos-Operator)가 감시하고 카오스-실험(Chaos-Experiments)을 호출합니다.
-- **ChaosExperiment**: 카오스 실험의 구성 파라미터를 그룹화하는 리소스입니다. ChaosExperiment CR은 ChaosEngine에서 실험을 호출할 때 오퍼레이터가 생성합니다.
-- **ChaosResult**: 카오스 실험의 결과를 보관할 리소스입니다. 카오스-익스포터(Chaos-exporter)는 결과를 읽고 구성된 프로메테우스(Prometheus) 서버로 메트릭을 내보냅니다.
+ChaosExperiment 및 ChaosEngine CR은 Workflow 객체에 포함되며 이는 하나 이상의 실험을 원하는 순서대로 연결할 수 있습니다.
 
-카오스 실험은 <a href="https://hub.litmuschaos.io" target="_blank">hub.litmuschaos.io</a>에서 호스팅합니다. 이를 통해 사용자가 프로덕션 애플리케이션의 회복 탄력성을 높일 수 있도록 애플리케이션 개발자 또는 공급 업체가 카오스 실험을 공유하는 중앙 허브입니다.
+- **ChaosResult**: A resource to hold the results of the experiment run. It provides details of the success of each validation constraint, the revert/rollback status of the fault as well as a verdict. The Chaos-exporter reads the results and exposes information as prometheus metrics. ChaosResults are especially useful during automated runs. 카오스 실험 실행 결과를 저장하는 리소스입니다. 각 검증 제약 조건의 성공 여부, 결함의 복구/롤백 상태 및 판결에 대한 세부 정보를 제공합니다. Chaos-exporter는 결과 정보를 Prometheus 메트릭으로 보여줍니다. ChaosResults는 자동화된 실행 중에 특히 유용합니다.
```
[the remark this answers is not in the document]
Comment (suhyenim) [reply translations/README-ko.md]: Sure, I will check the reviews and commit the changes. Thank you :)

**Review by namkyu1999 — APPROVED** [review]
LGTM 🚀

**Review by jongwooo — COMMENTED** [review]
nit: It would be nice to have a Korean glossary. (e.g., [Kubernetes Korean glossary](https://kubernetes.io/ko/docs/contribute/localization_ko/#한글화-용어집-정보))

**Comment by neelanjan00** [comment]
@namkyu1999 are we good to merge this PR?

**Comment by namkyu1999** [comment]
> @namkyu1999 are we good to merge this PR?

Yes, looks good to me

**Review by Saranya-jena — APPROVED** [review]

**Pull request merged**